csharp/src/Ice/Internal/ConnectionFactory.cs

```diff
@@ -1365,7 +1365,7 @@ public override void message(ThreadPoolCurrent current)
                         {
                             _instance.initializationData().logger.trace(
                                 _instance.traceLevels().networkCat,
-                                $"rejecting new {_endpoint.protocol()} connection at {_acceptor} because the maximum number of connections has been reached");
+                                $"rejecting new {_endpoint.protocol()} connection\n{transceiver}\nbecause the maximum number of connections has been reached");
```

Comment:
Adjust the tracing message to provide details about the transport connection being aborted.

Unfortunately, transceiver.ToString() is a multi-line description.